chain-core/src/common/fixed.rs

```diff
@@ -25,45 +26,196 @@ fn continued_fraction(
 }
 
 // https://en.wikipedia.org/wiki/Exponential_function#Continued_fractions_for_ex
-fn exp_series(
+fn exp2_series(
     x: FixedNumber,
+    y: FixedNumber,
 ) -> (
     impl Iterator<Item = FixedNumber>,
     impl Iterator<Item = FixedNumber>,
 ) {
-    let it_a = vec![FixedNumber::from_num(1), FixedNumber::from_num(1)]
+    let x2 = x * x;
+    let it_a = vec![FixedNumber::from_num(1), y * 2 - x]
         .into_iter()
-        .chain((2..).map(move |n| x + FixedNumber::from_num(n)));
-    let it_b = iter::once(1).chain((1..).map(|n| -n)).map(move |n| x * n);
+        .chain((0..).map(move |i| y * (i * 4 + 6)));
```

Comment:
I find this a bit difficult to unpack / reason about  -- perhaps this sub-expression `(0..).map(move |i| y * (i * 4 + 6))` could be taken out and given some name?

One other perhaps helpful thing for me would be if a (top-level) comment or a unit test showed the first N expected expressions in each iterator.
is it this?
`it_a = [1, y * 2 - x, 6*y, 10*y, 14*y ....]`
`it_b = [2*x, x^2, x^2 ... ]`
in order to relate that to the wiki article

Reply:
Added this comment.